### DISCUSSION TITLE
Welcome to leaflet-geosearch Discussions!

### DISCUSSION THREAD

**Discussion by smeijer** [smeijer/leaflet-geosearch#255] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nobbiexxx** [comment]
Hi

I'm using geosearch with Leaflet Js
I have included the script after the Leaflet script
and this

		new GeoSearch.GeoSearchControl({
			provider: new GeoSearch.OpenStreetMapProvider(),
		}).addTo(map);


The search works fine and produces results but when I click on a search result I get this in the console.

SearchControl.ts:412 Uncaught (in promise) TypeError: Cannot convert undefined or null to object
    at Function.keys (<anonymous>)
    at i.showResult (SearchControl.ts:412:28)
    at SearchControl.ts:404:12

Any help much appreciated

**Comment by andrealvesaraujo** [comment]
Great job to everyone.

**Comment by nobbiexxx** [comment]
I'm not sure I understand, I haven't been able to resolve this so have decide to take another route
